<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="19169" started-at="2018-10-04T05:30:50Z" finished-at="2018-10-04T05:31:09Z">
    <groups>
      <group name="regression">
        <method signature="LoginTest.invalidLoginTest_InvalidUserNameInvalidPassword(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="invalidLoginTest_InvalidUserNameInvalidPassword" class="com.leadgain.testcase.LoginTest"/>
        <method signature="LoginTest.validLoginTest_ValidUserNameValidPassword(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="validLoginTest_ValidUserNameValidPassword" class="com.leadgain.testcase.LoginTest"/>
      </group> <!-- regression -->
      <group name="smoke">
        <method signature="LoginTest.invalidLoginTest_InvalidUserNameInvalidPassword(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="invalidLoginTest_InvalidUserNameInvalidPassword" class="com.leadgain.testcase.LoginTest"/>
        <method signature="LoginTest.validLoginTest_ValidUserNameValidPassword(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="validLoginTest_ValidUserNameValidPassword" class="com.leadgain.testcase.LoginTest"/>
      </group> <!-- smoke -->
    </groups>
    <test name="Default test" duration-ms="19169" started-at="2018-10-04T05:30:50Z" finished-at="2018-10-04T05:31:09Z">
      <class name="com.leadgain.testcase.LoginTest">
        <test-method status="PASS" signature="springTestContextBeforeTestClass()[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="springTestContextBeforeTestClass" is-config="true" duration-ms="8" started-at="2018-10-04T05:30:50Z" finished-at="2018-10-04T05:30:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextBeforeTestClass -->
        <test-method status="PASS" signature="springTestContextPrepareTestInstance()[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="springTestContextPrepareTestInstance" is-config="true" duration-ms="8816" started-at="2018-10-04T05:30:50Z" depends-on-methods="org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestClass" finished-at="2018-10-04T05:30:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextPrepareTestInstance -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="setup" is-config="true" duration-ms="124" started-at="2018-10-04T05:30:58Z" description="Class Level Setup!" finished-at="2018-10-04T05:30:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="springTestContextBeforeTestMethod(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="springTestContextBeforeTestMethod" is-config="true" duration-ms="2" started-at="2018-10-04T05:30:59Z" finished-at="2018-10-04T05:30:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.leadgain.testcase.LoginTest.invalidLoginTest_InvalidUserNameInvalidPassword(java.lang.reflect.Method) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextBeforeTestMethod -->
        <test-method status="FAIL" signature="invalidLoginTest_InvalidUserNameInvalidPassword(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="invalidLoginTest_InvalidUserNameInvalidPassword" duration-ms="5776" started-at="2018-10-04T05:30:59Z" description="Invalid Login Scenario with wrong username and password." finished-at="2018-10-04T05:31:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.leadgain.testcase.LoginTest.invalidLoginTest_InvalidUserNameInvalidPassword(java.lang.reflect.Method) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Actual heading '' should be same as expected 'LEADGAIN'. expected [LEADGAIN] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Actual heading '' should be same as expected 'LEADGAIN'. expected [LEADGAIN] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.leadgain.page.LoginPage.verifyLoginPageHeader(LoginPage.java:63)
at com.leadgain.testcase.LoginTest.invalidLoginTest_InvalidUserNameInvalidPassword(LoginTest.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:230)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:180)
at org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:242)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:579)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginTest_InvalidUserNameInvalidPassword -->
        <test-method status="PASS" signature="springTestContextAfterTestMethod(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="springTestContextAfterTestMethod" is-config="true" duration-ms="1" started-at="2018-10-04T05:31:05Z" finished-at="2018-10-04T05:31:05Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.leadgain.testcase.LoginTest.invalidLoginTest_InvalidUserNameInvalidPassword(java.lang.reflect.Method) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextAfterTestMethod -->
        <test-method status="PASS" signature="springTestContextBeforeTestMethod(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="springTestContextBeforeTestMethod" is-config="true" duration-ms="1" started-at="2018-10-04T05:31:05Z" finished-at="2018-10-04T05:31:05Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.leadgain.testcase.LoginTest.validLoginTest_ValidUserNameValidPassword(java.lang.reflect.Method) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextBeforeTestMethod -->
        <test-method status="PASS" signature="validLoginTest_ValidUserNameValidPassword(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="validLoginTest_ValidUserNameValidPassword" duration-ms="557" started-at="2018-10-04T05:31:05Z" description="Valid Login Scenario with correct username and password." finished-at="2018-10-04T05:31:06Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.leadgain.testcase.LoginTest.validLoginTest_ValidUserNameValidPassword(java.lang.reflect.Method) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLoginTest_ValidUserNameValidPassword -->
        <test-method status="PASS" signature="springTestContextAfterTestMethod(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="springTestContextAfterTestMethod" is-config="true" duration-ms="0" started-at="2018-10-04T05:31:06Z" finished-at="2018-10-04T05:31:06Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.leadgain.testcase.LoginTest.validLoginTest_ValidUserNameValidPassword(java.lang.reflect.Method) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextAfterTestMethod -->
        <test-method status="PASS" signature="springTestContextBeforeTestMethod(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="springTestContextBeforeTestMethod" is-config="true" duration-ms="1" started-at="2018-10-04T05:31:06Z" finished-at="2018-10-04T05:31:06Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.leadgain.testcase.LoginTest.invalidLoginTest_EmptyUserEmptyPassword() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextBeforeTestMethod -->
        <test-method status="FAIL" signature="invalidLoginTest_EmptyUserEmptyPassword()[pri:1, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="invalidLoginTest_EmptyUserEmptyPassword" duration-ms="1858" started-at="2018-10-04T05:31:06Z" description="Invalid Login Scenario with empty username and password." finished-at="2018-10-04T05:31:08Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Actual 'Please Enter Your User Name' should be same as expected 'Please Enter Your Password'. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Actual 'Please Enter Your User Name' should be same as expected 'Please Enter Your Password'. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at com.leadgain.page.LoginPage.verifyLoginWithEmptyUser(LoginPage.java:104)
at com.leadgain.testcase.LoginTest.invalidLoginTest_EmptyUserEmptyPassword(LoginTest.java:81)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:230)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:180)
at org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:242)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:579)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginTest_EmptyUserEmptyPassword -->
        <test-method status="PASS" signature="springTestContextAfterTestMethod(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="springTestContextAfterTestMethod" is-config="true" duration-ms="0" started-at="2018-10-04T05:31:08Z" finished-at="2018-10-04T05:31:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.leadgain.testcase.LoginTest.invalidLoginTest_EmptyUserEmptyPassword() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextAfterTestMethod -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="teardown" is-config="true" duration-ms="587" started-at="2018-10-04T05:31:08Z" description="Class Level Teardown!" finished-at="2018-10-04T05:31:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method status="PASS" signature="springTestContextAfterTestClass()[pri:0, instance:com.leadgain.testcase.LoginTest@15405bd6]" name="springTestContextAfterTestClass" is-config="true" duration-ms="0" started-at="2018-10-04T05:31:09Z" finished-at="2018-10-04T05:31:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextAfterTestClass -->
      </class> <!-- com.leadgain.testcase.LoginTest -->
      <class name="com.leadgain.gui.testng.TestNGBase">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.leadgain.gui.testng.TestNGBase@33063f5b]" name="setUp" is-config="true" duration-ms="31" started-at="2018-10-04T05:30:49Z" finished-at="2018-10-04T05:30:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.leadgain.gui.testng.TestNGBase@33063f5b]" name="tearDown" is-config="true" duration-ms="1" started-at="2018-10-04T05:31:09Z" finished-at="2018-10-04T05:31:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.leadgain.gui.testng.TestNGBase -->
      <class name="com.leadgain.testcase.DashboardTest">
        <test-method status="PASS" signature="springTestContextBeforeTestClass()[pri:0, instance:com.leadgain.testcase.DashboardTest@352ed70d]" name="springTestContextBeforeTestClass" is-config="true" duration-ms="1" started-at="2018-10-04T05:31:06Z" finished-at="2018-10-04T05:31:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextBeforeTestClass -->
        <test-method status="PASS" signature="springTestContextPrepareTestInstance()[pri:0, instance:com.leadgain.testcase.DashboardTest@352ed70d]" name="springTestContextPrepareTestInstance" is-config="true" duration-ms="6" started-at="2018-10-04T05:31:06Z" depends-on-methods="org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestClass" finished-at="2018-10-04T05:31:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextPrepareTestInstance -->
        <test-method status="PASS" signature="springTestContextBeforeTestMethod(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.DashboardTest@352ed70d]" name="springTestContextBeforeTestMethod" is-config="true" duration-ms="1" started-at="2018-10-04T05:31:06Z" finished-at="2018-10-04T05:31:06Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.leadgain.testcase.DashboardTest.simpleTest()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextBeforeTestMethod -->
        <test-method status="PASS" signature="simpleTest()[pri:0, instance:com.leadgain.testcase.DashboardTest@352ed70d]" name="simpleTest" duration-ms="9" started-at="2018-10-04T05:31:06Z" finished-at="2018-10-04T05:31:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- simpleTest -->
        <test-method status="PASS" signature="springTestContextAfterTestMethod(java.lang.reflect.Method)[pri:0, instance:com.leadgain.testcase.DashboardTest@352ed70d]" name="springTestContextAfterTestMethod" is-config="true" duration-ms="1" started-at="2018-10-04T05:31:06Z" finished-at="2018-10-04T05:31:06Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.leadgain.testcase.DashboardTest.simpleTest()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextAfterTestMethod -->
        <test-method status="PASS" signature="springTestContextAfterTestClass()[pri:0, instance:com.leadgain.testcase.DashboardTest@352ed70d]" name="springTestContextAfterTestClass" is-config="true" duration-ms="0" started-at="2018-10-04T05:31:06Z" finished-at="2018-10-04T05:31:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextAfterTestClass -->
      </class> <!-- com.leadgain.testcase.DashboardTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
